cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/datastore/Database200MigrationTest.java

```diff
@@ -100,7 +102,7 @@ public Void call(SQLDatabase db) throws Exception {
 
 
         int count = revisionCount();
-        Assert.assertThat(count, is(2));
+        Assert.assertThat("That a duplicate root has been removed.", count, is(2));
```

Comment:
This message reads really nicely in the code, but it makes for a really confusing failure message because it makes it sound like the failure is because the duplicate has been removed not because it hasn't.

>java.lang.AssertionError: That a duplicate root has been removed.
Expected: is <2>
     but: was <3>

This is why I recommended using "should" because it tends to read OK both ways e.g.

>java.lang.AssertionError: The duplicate root should be removed.
Expected: is <2>
     but: was <3>

`Assert.assertThat("The duplicate root should be removed.", count, is(2)`

Do you see what I mean?